llvm/test/CodeGen/StubV/sra.mir

```diff
@@ -0,0 +1,78 @@
+# NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+# RUN: llc %s -mtriple=stubv32 -start-after=unpack-mi-bundles -o - | FileCheck %s
+
+--- |
+  ; ModuleID = '<stdin>'
+  source_filename = "<stdin>"
+  target datalayout = "e-m:e-p:32:32-i64:64-n32-S128"
+  target triple = "stubv32"
+
+  define i32 @foo(i32 %a, i32 %b) {
+  ; CHECK-LABEL: foo:
+  ; CHECK:       # %bb.0:
+  ; CHECK-NEXT:    sra a0, a0, a1
+  ; CHECK-NEXT:    ret
+    %c = ashr i32 %a, %b
```

Comment:
```suggestion
    %c = add i32 %a, %b
```

If I make this change the test still passes, when surely it should not?

Reply:
And if that means this test produces an `sra` for an LLVM IR `add`, why doesn't the add.mir test break with my change?